zendesk_api.gemspec

```diff
@@ -11,7 +11,7 @@ Gem::Specification.new do |s|
   s.homepage    = "https://developer.zendesk.com"
   s.summary     = 'Zendesk REST API Client'
   s.description = 'Ruby wrapper for the REST API at https://www.zendesk.com. Documentation at https://developer.zendesk.com.'
-  s.license     = 'Apache License Version 2.0'
+  s.license     = 'Apache-2.0'
```

Comment:
It is recommended in the [guide on Rubygems.org](https://guides.rubygems.org/specification-reference/#license=) to use the [SPDX identifier](https://spdx.org/licenses/) here.